packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -87,6 +88,20 @@ const COLUMN_DISK = {
     ),
   name: _('remoteDisk'),
 }
+const COLUMN_SPEED = {
+  itemRenderer: remote => {
+    const benchmark = get(() => remote.benchmarks[remote.benchmarks.length - 1])
+
+    return (
+      benchmark !== undefined && (
+        <span>{`${formatSize(benchmark.writeRate)}/s / ${formatSize(
```

Comment:
I think we should use `formatSpeed` here.

Even if it leads to add a useless `1e3` param, it will at least keep the speed formatting centralized in XO, which is good for maintenance.